identity/handler.go

```diff
@@ -234,29 +241,29 @@ type AdminCreateIdentityBody struct {
 
 // swagger:route POST /identities v0alpha2 adminCreateIdentity
 //
-// Create an Identity
+// # Create an Identity
 //
 // This endpoint creates an identity. It is NOT possible to set an identity's credentials (password, ...)
 // using this method! A way to achieve that will be introduced in the future.
 //
 // Learn how identities work in [Ory Kratos' User And Identity Model Documentation](https://www.ory.sh/docs/next/kratos/concepts/identity-user-model).
 //
-//     Consumes:
-//     - application/json
+//	    Consumes:
+//	    - application/json
 //
-//     Produces:
-//     - application/json
+//	    Produces:
+//	    - application/json
 //
-//     Schemes: http, https
+//	    Schemes: http, https
 //
-//     Security:
-//       oryAccessToken:
+//	    Security:
+//	      oryAccessToken:
 //
-//     Responses:
-//       201: identity
-//       400: jsonError
-//		 409: jsonError
-//       500: jsonError
+//	    Responses:
+//	      201: identity
+//	      400: jsonError
+//			 409: jsonError
```

Comment:
This line looks really odd. If it's a formatter, I'm now sure it does weird things.